shell/src/main/java/alluxio/cli/ConfigurationDocGenerator.java

```diff
@@ -125,6 +125,83 @@ public static void writeCSVFile(Collection<? extends PropertyKey> defaultKeys, S
   }
 
   /**
+   * Writes description of property key to yml files.
+   *
+   * @param defaultKeys Collection which is from PropertyKey DEFAULT_KEYS_MAP.values()
+   * @param filePath    path for csv files
+   */
+  public static void writeYMLFile(Collection<? extends PropertyKey> defaultKeys, String filePath)
+      throws IOException {
+    if (defaultKeys.size() == 0) {
+      return;
+    }
+
+    FileWriter fileWriter;
+
+    //HashMap for yml file names
+    Map<String, String> fileNames = new HashMap<>();
```

Comment:
Can we get rid of the `fileNames` map, and just create the `fileWriterMap` directly?